src/lib.rs

```
@@ -65,17 +66,30 @@ pub async fn threema_incoming_message_handler(
             Message::GroupTextMessage(group_text_msg) => {
                 let matrix_client = app_state.matrix_client.lock().await;
 
-                let content = RoomMessageEventContent::text_plain(
-                    group_text_msg.base.push_from_name.unwrap()
-                        + ": "
-                        + group_text_msg.text.as_str(),
-                );
-                let txn_id = TransactionId::new();
-
-                matrix_client.joined_rooms()[0]
-                    .send(content, Some(&txn_id))
-                    .await
-                    .unwrap();
+                if group_text_msg.text.starts_with("!threematrix") {
+                    let splittet_text: Vec<&str> = group_text_msg.text.split(" ").collect();
```

We should add a check whether the result has exactly two elements. If it's shorter (e.g. "!threematrix" or longer "!threematrix one two"), we should throw an error. But this should probably be its own "validation" ticket.